src/DiffEqSensitivity.jl

```diff
@@ -18,13 +18,14 @@ include("morris_sensitivity.jl")
 include("sobol_sensitivity.jl")
 include("regression_sensitivity.jl")
 include("DGSM.jl")
+include("eFAST.jl")
```

Comment:
is this from the wrong branch?

Reply:
Yes, just this file though. Will fix in next commit.